src/map/map.hpp

```diff
@@ -334,7 +334,7 @@ enum e_race : int8{
 };
 
 enum e_race2 : uint8{
-	RC2_NONE = 0,
+	RC2_NONE = 0,	// Allows to remove RaceGroups in import
```

Comment:
```suggestion
	RC2_NONE = 0,
```
This does not allow anything